cli/azd/internal/cmd/add/add_select_ai.go

```diff
@@ -0,0 +1,192 @@
+		if err != nil {
+			return nil, fmt.Errorf("failed creating HTTP pipeline: %w", err)
+		}
+
+		console.ShowSpinner(
+			ctx,
+			fmt.Sprintf("Fetching available models in %s...", a.env.GetLocation()),
+			input.Step)
+
+		location := fmt.Sprintf(
+			//nolint:lll
+			"https://management.azure.com/subscriptions/%s/providers/Microsoft.CognitiveServices/locations/%s/models?api-version=2023-05-01",
+			a.env.GetSubscriptionId(),
+			a.env.GetLocation())
+		req, err := runtime.NewRequest(ctx, http.MethodGet, location)
+		if err != nil {
+			return nil, fmt.Errorf("creating request: %w", err)
+		}
+
+		resp, err := pipeline.Do(req)
+		if err != nil {
+			return nil, fmt.Errorf("making request: %w", err)
+		}
+
+		if resp.StatusCode != http.StatusOK {
+			return nil, runtime.NewResponseError(resp)
+		}
+
+		body, err := runtime.Payload(resp)
+		if err != nil {
+			return nil, fmt.Errorf("reading response: %w", err)
+		}
+
+		console.StopSpinner(ctx, "", input.Step)
+		var response ModelResponse
+		err = json.Unmarshal(body, &response)
+		if err != nil {
+			return nil, fmt.Errorf("decoding response: %w", err)
+		}
+
+		for _, model := range response.Value {
+			if model.Kind == "OpenAI" && slices.ContainsFunc(model.Model.Skus, func(sku ModelSku) bool {
+				return sku.Name == "Standard"
+			}) {
+				switch aiOption {
+				case 0:
+					if model.Model.Name == "gpt-4o" || model.Model.Name == "gpt-4" {
+						allModels = append(allModels, model)
+					}
+				case 1:
+					if strings.HasPrefix(model.Model.Name, "text-embedding") {
+						allModels = append(allModels, model)
+					}
+				}
+			}
+
+		}
+		if len(allModels) > 0 {
+			break
+		}
+
+		_, err = a.rm.FindResourceGroupForEnvironment(
+			ctx, a.env.GetSubscriptionId(), a.env.Name())
+		var notFoundError *azureutil.ResourceNotFoundError
+		if errors.As(err, &notFoundError) { // not yet provisioned, we're safe here
+			console.MessageUxItem(ctx, &ux.WarningMessage{
+				Description: fmt.Sprintf("No models found in %s", a.env.GetLocation()),
+			})
+			confirm, err := console.Confirm(ctx, input.ConsoleOptions{
+				Message: "Try a different location?",
+			})
+			if err != nil {
+				return nil, err
+			}
+			if confirm {
+				a.env.SetLocation("")
+				continue
+			}
+		} else if err != nil {
+			return nil, fmt.Errorf("finding resource group: %w", err)
+		}
+
+		return nil, fmt.Errorf("no models found in %s", a.env.GetLocation())
+	}
+
+	slices.SortFunc(allModels, func(a ModelList, b ModelList) int {
+		return strings.Compare(b.Model.SystemData.CreatedAt, a.Model.SystemData.CreatedAt)
+	})
+
+	displayModels := make([]string, 0, len(allModels))
+	models := make([]Model, 0, len(allModels))
+	for _, model := range allModels {
+		models = append(models, model.Model)
+		displayModels = append(displayModels, fmt.Sprintf("%s\t%s", model.Model.Name, model.Model.Version))
+	}
+
+	if console.IsSpinnerInteractive() {
+		displayModels, err = output.TabAlign(displayModels, 5)
```

Comment:
Just wondering - in the other two calls to `output.TabAlign` we passed `3` not `5`.  Was it intended that this is more than the others?

Reply:
It was an intentional change that does feel better here.

The other texts have a bracket label for one of the columns, i.e.:

`[Container App] api`

which feels good.

Here we have

`gpt-4o     2023-04-11` 

which feels better with more space